src/main/java/org/launchcode/recipeapp/controllers/HomeController.java

```diff
@@ -1,12 +1,13 @@
 package org.launchcode.recipeapp.controllers;
 
+import org.springframework.stereotype.Controller;
```

Comment:
This one isn't being used, so you don't really need it.

Reply:
Agree, try to remove unused imports where possible.